docs/adr/0005-graduation of EventBridge.md

```diff
@@ -0,0 +1,57 @@
+# 5. Graduation of EventBridge in CAPA
+
+* Status: [proposed]
+* Date: TBD 
+* Authors: @Ankitasw
+* Deciders: @richardcase @sedefsavas
+
+## Context
+EventBridge is an eventbus provided by AWS to watch for the events generated by any SaaS application running on AWS, or any of the AWS services, that could be used by any other services.
+
+With respect to CAPA, EventBridge helps watch EC2 instance state change events using which _AWSMachine_ controller would trigger reconciliation and handle the lifecycle of the EC2 instances.
```

Comment:
```suggestion
With respect to CAPA, the  _AWSMachine_ controller uses EC2 instance state change events via EventBridge to  trigger reconciliation so that it can handle the lifecycle of the EC2 instances.
```